homeassistant/components/insteon.py

```diff
@@ -30,15 +29,26 @@ def setup(hass, config):
     """
     if not validate_config(
             config,
-            {DOMAIN: [CONF_USERNAME, CONF_PASSWORD]},
+            {DOMAIN: [CONF_USERNAME, CONF_PASSWORD, CONF_API_KEY]},
             _LOGGER):
         return False
 
-    import insteon
+    try:
```

Comment:
Ahh, ok, got it. I saw this pattern in [isy994.py](https://github.com/balloob/home-assistant/blob/dev/homeassistant/components/isy994.py#L41) so I assumed it was the right approach. I'll remove the try if it's not needed.